sensor/kubernetes/listener/resources/networkpolicy_test.go

```diff
@@ -0,0 +1,445 @@
+package resources
+
+import (
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/stackrox/rox/generated/internalapi/central"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/features"
+	"github.com/stackrox/rox/sensor/common/detector/mocks"
+	"github.com/stretchr/testify/assert"
+	"k8s.io/apimachinery/pkg/labels"
+)
+
+func TestGetSelector(t *testing.T) {
+	if !features.NetworkPolicySystemPolicy.Enabled() {
```

Comment:
With the way this is now, it seems like the tests aren't running at all in the CI. Maybe add a `SetupTest()` in this setting the feature flag to true so at least we get tests running in `go-unit-tests` step. You can keep this check here to avoid running the tests in release mode.

Reply:
Done. I've changed the tests to use testify and now everything should be set in `SetupTest()`